branchpro/apps/_simulation.py

```diff
@@ -283,6 +294,36 @@ def get_sliders_ids(self):
         """
         return self.sliders.slider_ids()
 
+    def add_simulation(self):
+        """
+        Adds new simulated graph in the figure for the current slider values.
+        """
+        data = self.simulator.run(self._init_cond)
+        df = pd.DataFrame({
+            'Time': self.simulator.get_regime(),
+            'Incidence Number': data})
+
+        self.plot.add_simulation(df)
+
+        return self.plot.figure
+
+    def clear_simulations(self):
+        """
+        Clears all simulations currently plotted in the figure.
```

Comment:
I would mention in the docstring that this method also adds one new simulation after clearing.